README

```diff
@@ -155,6 +155,7 @@ CHECK IDS
 	E-004: Nonexistent role listed in fc file
 	E-005: Nonexistent type listed in fc file
 	E-006: Attribute and interface with same name
+	E-007: Module name does not match file name
```

Comment:
I feel like this should be a warning instead of an error.

Reply:
I think `checkpolicy(8)` treats it as error, see https://github.com/SELinuxProject/selinux/blob/9d9a3307de550a4ba31ba41a5e71f3ac27423dbb/checkpolicy/checkmodule.c#L303

Reply:
Interesting.  That's actually good to know.  Last time I encountered this situation must have been before 2016.

In refpolicy builds with modules in base though, the module name will just be discarded, so this check will never be reached.  That feels similar to most of the existing warnings that can cause build time errors in some scenarios but not others.

Reply:
changed to severity warning